deployment/cdk/opensearch-service-migration/lib/common-utilities.ts

```diff
@@ -435,3 +439,27 @@ export function isStackInGovCloud(stack: Stack): boolean {
 export function isRegionGovCloud(region: string): boolean {
     return region.startsWith('us-gov-');
 }
+
+/**
+ * Creates a Docker image asset from the specified image name.
+ *
+ * This function creates a Docker image asset from the specified image name.
+ * It can use a local or remote image.
+ *
+ * @param {string} imageName - The name of the Docker image to use as the base image.
+ * @returns {ContainerImage} - A `ContainerImage` object representing the Docker image asset.
+ */
+export function makeDockerImageAsset(scope: Construct, serviceName: string, imageName: string): ContainerImage {
```

Comment:
The other benefit here is that we're not depending on the build output directory anymore. I don't know of any functional difference with the wrapper layer being here

Reply:
This is just an alias, right?  Please rename the function & explain why we need an alias.

Reply:
Updated, this is an alias, but it also pulls the image locally. This will add it to a private ECR instead of pulling on container start potentially from an outside source (which can add costly nat gateway bills)